base/nullable.jl

```diff
@@ -53,20 +53,20 @@ otherwise, returns `y` if provided, or throws a `NullException` if not.
 """
 @inline function get{S,T}(x::Nullable{S}, y::T)
     if isbits(S)
-        ifelse(x.isnull, y, x.value)
+        ifelse(isnull(x), y, x.value)
```

Comment:
Not a bad idea in general, but I don't think we have any benchmarks for `Nullable` yet. We really need to add some. We could also check the generated code manually in some cases.